### PR TITLE
[Merged by Bors] - Updated docs

### PR DESCRIPTION
**Pull request by fhennig**
# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/trino/pages/concepts.adoc [hunk 1]
```diff
@@ -1,13 +1,13 @@
 = Concepts
 
-== Connectors
+== [[connectors]]Connectors
 
 https://trino.io/docs/current/overview/use-cases.html#what-trino-is[Trino] is a tool designed to efficiently query vast amounts of data using distributed queries. It is not a database with its own store but rather interacts with many types of store. Trino connects to these stores - or data sources - via https://trino.io/docs/current/connector.html[connectors].
 Each connector enables access to a specific underlying datasource such as a Hive warehouse, a PostgreSQL database or a Druid instance.
 
 A Trino cluster comprises two roles: the Coordinator, responsible for managing and monitoring work loads, and the Worker, which is responsible for executing specific tasks that together make up a work load. The workers fetch data from the connectors, execute tasks and share intermediate results. The coordinator collects and consolidates these results for the end-user.
```
Comment (maltesander) [inline docs/modules/trino/pages/concepts.adoc:8]: Not part of this PR, just did not like the term "store".
```suggestion
https://trino.io/docs/current/overview/use-cases.html#what-trino-is[Trino] is a tool designed to efficiently query vast amounts of data using distributed queries. It is not a database with its own storage but rather interacts with many different data stores. Trino connects to these data stores - or data sources - via https://trino.io/docs/current/connector.html[connectors].
Each connector enables access to a specific underlying data source such as a Hive warehouse, a PostgreSQL database or a Druid instance.

A Trino cluster comprises two roles: the Coordinator, responsible for managing and monitoring work loads, and the Worker, which is responsible for executing specific tasks that together make up a work load. The workers fetch data from the connectors, execute tasks and share intermediate results. The coordinator collects and consolidates these results for the end-user.
```

File: docs/modules/trino/pages/index.adoc [hunk 2]
```diff
@@ -1,18 +1,42 @@
 = Stackable Operator for Trino
+:description: The Stackable Operator for Trino is a Kubernetes operator that can manage Trino clusters. Learn about its features, resources, dependencies and demos, and see the list of supported Trino versions.
+:keywords: Stackable Operator, Trino, Kubernetes, k8s, operator, data science, data exploration, SQL, engineer, big data, CRD, StatefulSet, ConfigMap, Service, Druid, Trino, S3, Superset
 
 This is an operator for Kubernetes that can manage https://https://trino.io/[Trino] clusters.
+Trino is an open-source distributed SQL query engine that enables high-speed analytics of large datasets from multiple data sources using SQL queries.This operator enables you to manage your Trino instances on Kubernetes efficiently.
```
Comment (maltesander) [inline docs/modules/trino/pages/index.adoc:6]:
```suggestion
Trino is an open-source distributed SQL query engine that enables high-speed analytics of large datasets from multiple data sources using SQL queries. This operator enables you to manage your Trino instances on Kubernetes efficiently.
```

File: docs/modules/trino/pages/usage_guide/s3.adoc [hunk 3]
```diff
@@ -0,0 +1,88 @@
+= S3 connection specification
+
+You can specify S3 connection details directly inside the TrinoCatalog specification
+or by referring to an external`S3Connection custom resource.
```
Comment (maltesander) [inline docs/modules/trino/pages/usage_guide/s3.adoc:4]: Maybe link to the S3Connection concepts page?
```suggestion
or by referring to an external S3Connection custom resource.
```

**Review by maltesander — CHANGES_REQUESTED** [review]
Minor things. LGTM otherwise!

File: docs/modules/trino/pages/usage_guide/security.adoc [hunk 4]
```diff
@@ -62,4 +62,76 @@ In order to authorize Trino via OPA, a `ConfigMap` containing Rego rules for Tri
...
+<2> The name of (and reference to) the `Secret`
+<3> `TrinoCatalog` reference
+<4> TLS mechanism
+
+The CLI now requires that a path to the keystore and a password be provided:
+
+[source]
+----
+./trino.jar --debug --server https://172.18.0.3:31748
+--user=admin --keystore-path=<path-to-keystore.p12> --keystore-password=<password>
```
[the remark this answers is not in the document]
Comment (fhennig) [reply docs/modules/trino/pages/usage_guide/security.adoc:96]: It's already in a `[source]` block on a separate line

**Review by maltesander — APPROVED** [review]
LGTM!

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**